amqp-postgres/amqp_postgres/postgres_publisher.py

```diff
@@ -197,9 +197,9 @@ def _message_publisher(self):
                     self._store(conn, items)
                 except psycopg2.OperationalError as e:
                     self.on_db_connection_error(e)
-                except (psycopg2.IntegrityError, ValueError):
-                    logger.exception('Error storing %d logs+events',
-                                     len(items))
+                except Exception:
+                    logger.info('Error storing %d logs+events in batch',
```

Comment:
any exception will make this go into the no-batch mode; the errors are then analyzed there, if they happen again, no need to print the whole traceback here